config/internal/configsource/manager.go

```diff
@@ -91,6 +100,21 @@ import (
 //      # as a string if params doesn't specify that "binary" is true.
 //      text_from_file: $file:/etc/text.txt
 //
+// Bracketed single-line should be used when concatenating a suffix to the value retrieved by
+// the config source. Example:
+//
+//    component:
+//      # Retrieves the value of the environment variable LOGS_DIR and appends /component.log to it.
+//      log_file_fullname: ${env:LOGS_DIR}/component.log
```

Comment:
Is `env` already an existing config source or this is just an example of a possible future functionality that will exist once we implement `env` as a config source?

Reply:
It does not exist yet it is just a possible future implementation. If having it as an example is confusing I can change it to something else.

Reply:
Reading the text from the top there is already a mention that `env` is a hypothetical config source.